cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -1,16 +1,16 @@
 NAME
-    Ballerina build - Builds Ballerina module(s) and generates executable
+    Ballerina build - Builds a Ballerina module(s) and generates executable
```

Comment:
Doesn't "a" sound odd with "module(s)"?

Reply:
```suggestion
    Ballerina build - Builds a Ballerina module(s) and generates executables
```

Reply:
Priority is given to the word "module" here ("s" is within brackets) and that needs the "a".

Reply:
It is correct to say "executable outputs."